packages/blue-sdk/src/market/Market.ts

```diff
@@ -378,8 +383,13 @@ export class Market implements InputMarket {
    * Returns the maximum amount of loan assets that can be borrowed given a certain amount of collateral.
    * @param collateral The amount of collateral to consider.
    */
-  public getMaxBorrowAssets(collateral: bigint) {
-    return MarketUtils.getMaxBorrowAssets(collateral, this, this.config);
+  public getMaxBorrowAssets(
+    collateral: bigint,
+    options: MaxCapacitiesOptions["borrow"] = {},
```

Comment:
```suggestion
    { lltv = this.config.lltv }: { lltv?: bigint } = {},
```

Reply:
I prefer keeping the naming `lltv` because `maxLtv` may suggest that the resulting ltv may be lower than the given `maxLtv`

Reply:
This isn't technically a lltv 🤔  so this naming would be wrong

The "max" here refers to the "max" in the name of the fonction: if the max ltv is X, you can borrow up to Y

Reply:
Ok then `ltv`

Reply:
It is technically a lltv (liquidation ltv): max borrow assets is only defined given an amount of collateral and a liquidation ltv

If given an amount of collateral and a ltv, then a better naming is "borrow assets" and not "max borrow assets" (or "borrowable" or whatever, but it's not a maximum anymore)

Reply:
Well it is the maximum borrowable assets given a maximum ltv 🤔 I don't get your point

Reply:
max ltv == liquidation ltv, but max ltv is not a naming used in the ecosystem

Reply:
In the end I'm ok with `maxLtv`